cypress/e2e/tests/pages/manager/cluster-provisioning-azure-rke1.spec.ts

```diff
@@ -0,0 +1,254 @@
+import HomePagePo from '@/cypress/e2e/po/pages/home.po';
+import ClusterManagerCreateRke1AzurePagePo from '@/cypress/e2e/po/edit/provisioning.cattle.io.cluster/create/cluster-create-rke1-azure.po';
+import ClusterManagerListPagePo from '@/cypress/e2e/po/pages/cluster-manager/cluster-manager-list.po';
+import ClusterManagerDetailRke1AzurePagePo from '@/cypress/e2e/po/detail/provisioning.cattle.io.cluster/cluster-detail-rke1-azure.po';
+import PromptRemove from '@/cypress/e2e/po/prompts/promptRemove.po';
+import LoadingPo from '@/cypress/e2e/po/components/loading.po';
+import EmberBannersPo from '@/cypress/e2e/po/components/ember/ember-banners.po';
+
+// will only run this in jenkins pipeline where cloud credentials are stored
+describe('Provision Node driver RKE1 cluster with Azure', { testIsolation: 'off', tags: ['@manager', '@adminUser', '@standardUser', '@jenkins'] }, () => {
+  const clusterList = new ClusterManagerListPagePo();
+  const createRKE1ClusterPage = new ClusterManagerCreateRke1AzurePagePo();
+  let removeNodeTemplate = false;
+  let cloudcredentialId = '';
+  let nodeTemplateId = '';
+
+  before(() => {
+    cy.login();
+    HomePagePo.goTo();
+
+    // clean up azure node templates and cloud credentials
```

Comment:
Very nitpicky request, could this and below be made very prominent, just in case someone runs this locally and finds their test cluster changed? maybe something at the top of the file like below
```
/******
 *  Running this test will delete x and y resources from the target cluster
 ******/
 ```

Reply:
good call!